plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/env/PydevIProcessFactory.java

```diff
@@ -169,19 +170,24 @@ public PydevConsoleLaunchInfo createInteractiveLaunch() throws UserCanceledExcep
             }
 
             if (interpreter == null) {
-                SelectionDialog listDialog = AbstractInterpreterPreferencesPage.createChooseIntepreterInfoDialog(
-                        workbenchWindow, interpreters, "Select interpreter to be used.", false);
-
-                int open = listDialog.open();
-                if (open != ListDialog.OK || listDialog.getResult().length > 1) {
-                    return null;
-                }
-                Object[] result = listDialog.getResult();
-                if (result == null || result.length == 0) {
-                    interpreter = interpreters[0];
-
+                if (PydevConsoleConstants.ACTIVE_EDITOR_INTERPRETER_REPRESENTATION
```

Comment:
Please check if `edit != null` (if the current editor is not a PyDev editor, or if there's no current editor, this will fail).